docs/features/configuration.rst

```diff
@@ -132,6 +132,16 @@ You can also give Ocelot a specific path to look in for the configuration files
 
 Ocelot needs the HostingEnvironment so it knows to exclude anything environment specific from the algorithm. 
 
+Merging configuration files to memory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default Ocelot writes the merged configuration back to disk as ``ocelot.json`` and add that file to config.

```

Comment:
We need to show the default option of `MergeOcelotJson` enumeration, and explain to developers that default option is reused implicitly.